xbmc/pvr/PVRGUIActions.cpp

```diff
@@ -1502,10 +1502,14 @@ namespace PVR
 
     if (!bResetEPGOnly)
     {
-      pvrDatabase->DeleteChannelGroups();
+      pvrDatabase->DeleteClients();
```

Comment:
Is it correct to delete clients before channel groups?

Reply:
Well, technically it's okay but you might be right, because it feels wrong from the logical point of view. Will change the order. ;-)